guide/english/designer-tools/Inkscape/index.md

```diff
@@ -0,0 +1,32 @@
+---
+
+title:
+Inkscape
+---
+
+## Inkscape
+
+Inkscape is an open-source vector-based graphic editing tool used by graphic designers, artists andillustrationist to create high quality,
+vector-based illustration and designs. Inkscape is widely termed as 'free and open-source' alternative to popular illustration design tool Adobe Illustrator.
+
+Inkscape can be used to create or edit vector based graphics including illustrations, shapes, line arts, charts,logos, diagrams, 
+as well as raster images.
+
+Being an open-source tool, Inkscape comes free of cost. 
+
+You can freely download Inkscape [here](https://inkscape.org/release/0.92.3/)
```

Comment:
This line can be removed as there is a better download section which you have added down below